hs628_tools/Project1_Vizualization.py

```diff
@@ -0,0 +1,297 @@
+
+# coding: utf-8
+
+# In[20]:
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
```

Comment:
should never have the credentials displayed in your code, you can delete this line.

Reply:
okay thanks Rohit , appreciate it!